packages/medusa/src/api/routes/admin/customer-groups/index.ts

```diff
@@ -10,6 +10,10 @@ export default (app) => {
 
   route.get("/:id", middlewares.wrap(require("./get-customer-group").default))
   route.post("/", middlewares.wrap(require("./create-customer-group").default))
+  route.delete(
+    "/:id",
+    middlewares.wrap(require("./delete-customer-group").default)
+  )
   return app
 }
 
```

Comment:
Would be great to add `export type AdminCustomerGroupsDeleteRes = DeleteResponse` for when we add the endpoints in `medusa-js` 👍